cache/class-vary-cache.php

```diff
@@ -361,7 +362,8 @@ public static function is_encryption_enabled() {
 	private static function encrypt_cookie_value( $value ) {
 		$client_key = constant( 'VIP_GO_AUTH_COOKIE_KEY' );
 		$client_iv = constant( 'VIP_GO_AUTH_COOKIE_IV' );
-		$cookie_value = random_bytes( 32 ) . '|' . $value . '|' . ( time() + self::$cookie_expiry );
+		$random = random_int( 0,  PHP_INT_MAX ) . random_int( 0,  PHP_INT_MAX );
```

Comment:
Can we add some more context around why we're generating the random this way?

Reply:
I simplified this, I think trying to duplicate the 32 byte random characters was overkill